packages/backend-plugin-api/api-report.md

```diff
@@ -183,6 +242,7 @@ export namespace coreServices {
   const httpRouter: ServiceRef<HttpRouterService, 'plugin', 'singleton'>;
   const lifecycle: ServiceRef<LifecycleService, 'plugin', 'singleton'>;
   const logger: ServiceRef<LoggerService, 'plugin', 'singleton'>;
+  const eventAuditor: ServiceRef<EventAuditorService, 'plugin', 'singleton'>;
```

Comment:
Open discussion on naming: `auditor` and `rootAuditor` (skipping "event")? How do you and the maintainers feel about that? Not sure the "event" word adds anything.

Reply:
Let's go with `Auditor` for now since it's less overhead.